pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -628,20 +626,20 @@ func (service *SecurityPolicyService) buildRuleDisplayName(rule *v1alpha1.Securi
 	if ruleDirection == "IN" {
```

Comment:
Is this more clean?

```
	var ruleAct string
	switch ruleAction {
	case util.ToUpper(v1alpha1.RuleActionAllow):
		ruleAct = common.RuleActionAllow
	case util.ToUpper(v1alpha1.RuleActionDrop):
		ruleAct = common.RuleActionDrop
	case util.ToUpper(v1alpha1.RuleActionReject):
		ruleAct = common.RuleActionRejec
	}
	ruleDir := common.RuleEgress
	if ruleDirection == "IN" {
		ruleDir = common.RuleIngress
	}
	suffix := strings.Join([]string{ruleDir, ruleAct}, common.ConnectorUnderline)
```

Reply:
Thanks so much for suggestion, I refine the code as suggested, please take a look.